### DISCUSSION TITLE
Setting a bpm

### DISCUSSION THREAD

**Discussion by kad1kad** [danigb/smplr#13] [Q&A]
Hi,
Loving smplr so far. I want to build a step sequencer with it and was wondering whether there is a way to define a global bpm value.

**Comment by danigb** [comment]
Hi, thanks! This project only contains the instruments, not the scheduler. Currently the only way to play notes on time is doing calculations by yourself (using current time, bpm and note durations) or using external scheduler like https://github.com/sebpiq/WAAClock or similar.
